workflow/templateresolution/context.go

```diff
@@ -51,6 +51,13 @@ func WrapClusterWorkflowTemplateInterface(clusterClientset v1alpha1.ClusterWorkf
 	return &clusterWorkflowTemplateInterfaceWrapper{clientset: clusterClientset}
 }
 
+type NullClusterWorkflowTemplateGetter struct{}
```

Comment:
The pattern that we use here is to use "Explosive" for null objects... shall we continue using it? @alexec 
```suggestion
type ExplosiveClusterWorkflowTemplateGetter struct {
}
```

Reply:
not in this case - this is a inert implementation - "explosive" suggest something opposite